src/ZoneServer/Buffs/Handlers/Swordsman/Bear.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handle for the Bear Buff, which reduces damage taken
+	/// This class's effect is handled in calc_combat, but this handler
+	/// needs to remain so its duration can be saved on logout.
```

Comment:
Strange, the first time I tested it, it seemed like removing the Buffhandler made the buff not save, but this time it worked fine.  In that case I guess the Buffhandler can be removed, since it has no effect.